npm install fails with compilation error

# Operating system:
Debian Buster
- Node version:
18.14
- npm version:
9.3.1
- mediasoup version:
latest



### Issue description
When running npm install, fails with:
```
#12 177.3 npm ERR! [778/1349] Compiling C++ object mediasoup-worker.p/src_RTC_Transport.cpp.o
#12 177.3 npm ERR! FAILED: mediasoup-worker.p/src_RTC_Transport.cpp.o 
#12 177.3 npm ERR! c++ -Imediasoup-worker.p -I. -I../../.. -I../../../include -Isubprojects/abseil-cpp-20230802.0 -I../../../subprojects/abseil-cpp-20230802.0 -I../../../subprojects/openssl-3.0.8/include -I../../../subprojects/openssl-3.0.8/crypto -I../../../subprojects/openssl-3.0.8/crypto/modes -I../../../subprojects/openssl-3.0.8/crypto/ec/curve448 -I../../../subprojects/openssl-3.0.8/crypto/ec/curve448/arch_32 -I../../../subprojects/openssl-3.0.8/providers/common/include -I../../../subprojects/openssl-3.0.8/providers/implementations/include -Isubprojects/openssl-3.0.8/generated-config/archs/linux-x86_64/asm -I../../../subprojects/openssl-3.0.8/generated-config/archs/linux-x86_64/asm -I../../../subprojects/openssl-3.0.8/generated-config/archs/linux-x86_64/asm/include -I../../../subprojects/openssl-3.0.8/generated-config/archs/linux-x86_64/asm/crypto -I../../../subprojects/openssl-3.0.8/generated-config/archs/linux-x86_64/asm/providers/common/include -I../../../subprojects/libuv-v1.47.0/include -Isubprojects/libsrtp-2.5.0/include -I../../../subprojects/libsrtp-2.5.0/include -Isubprojects/usrsctp-ebb18adac6501bad4501b1f6dccb67a1c85cc299/usrsctplib -I../../../subprojects/usrsctp-ebb18adac6501bad4501b1f6dccb67a1c85cc299/usrsctplib -Isubprojects/usrsctp-ebb18adac6501bad4501b1f6dccb67a1c85cc299/usrsctplib/netinet -I../../../subprojects/usrsctp-ebb18adac6501bad4501b1f6dccb67a1c85cc299/usrsctplib/netinet -Isubprojects/usrsctp-ebb18adac6501bad4501b1f6dccb67a1c85cc299/usrsctplib/netinet6 -I../../../subprojects/usrsctp-ebb18adac6501bad4501b1f6dccb67a1c85cc299/usrsctplib/netinet6 -I../../../subprojects/flatbuffers-23.3.3/include -I../../../subprojects/flatbuffers-23.3.3/grpc -Ifbs -I../../../fbs -Ideps/libwebrtc -I../../../deps/libwebrtc -I../../../deps/libwebrtc/libwebrtc -Isubprojects/liburing-liburing-2.4/src/include -I../../../subprojects/liburing-liburing-2.4/src/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++17 -O3 -fPIE -pthread -DMS_LITTLE_ENDIAN -DMS_LIBURING_SUPPORTED -DMS_EXECUTABLE -MD -MQ mediasoup-worker.p/src_RTC_Transport.cpp.o -MF mediasoup-worker.p/src_RTC_Transport.cpp.o.d -o mediasoup-worker.p/src_RTC_Transport.cpp.o -c ../../../src/RTC/Transport.cpp
#12 177.3 npm ERR! In file included from ../../../subprojects/liburing-liburing-2.4/src/include/liburing.h:24,
#12 177.3 npm ERR!                  from ../../../include/DepLibUring.hpp:7,
#12 177.3 npm ERR!                  from ../../../src/RTC/Transport.cpp:7:
#12 177.3 npm ERR! subprojects/liburing-liburing-2.4/src/include/liburing/compat.h:5:10: fatal error: linux/time_types.h: No such file or directory
#12 177.3 npm ERR!  #include <linux/time_types.h>
#12 177.3 npm ERR!           ^~~~~~~~~~~~~~~~~~~~
#12 177.3 npm ERR! compilation terminated.
#12 177.3 npm ERR! [779/1349] Compiling C++ object mediasoup-worker.p/src_RTC_TcpServer.cpp.o

Probably irrelevant, but the prebuilt is failing so local build is done. This is a known issue with some not super recent Debian versions AFAIR:

```

npm-scripts.mjs [36m[INFO] [postinstall][0m downloadPrebuiltWorker() | got mediasoup-worker prebuilt binary
2023-12-04T17:08:36.0738600Z #12 177.3 npm ERR! npm-scripts.mjs [36m[INFO] [postinstall][0m downloadPrebuiltWorker() | checking fetched mediasoup-worker prebuilt binary in current host
2023-12-04T17:08:36.0740299Z #12 177.3 npm ERR! npm-scripts.mjs [36m[INFO] [postinstall][0m couldn't fetch any mediasoup-worker prebuilt binary, building it locally
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,11 @@
 # Changelog
 
 
+### NEXT
+
+* Update worker liburing dependency to 2.4-2 ([PR #XXXX](https://github.com/versatica/mediasoup/pull/XXXX)):
+
+
 ### 3.13.8
 
 * LibUring: Enable liburing usage for SCTP data delivery ([PR 1249](https://github.com/versatica/mediasoup/pull/1249)).

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,7 +3,7 @@
 
 ### NEXT
 
-* Update worker liburing dependency to 2.4-2 ([PR #XXXX](https://github.com/versatica/mediasoup/pull/XXXX)):
+* Update worker liburing dependency to 2.4-2 ([PR #1254](https://github.com/versatica/mediasoup/pull/1254)):
 
 
 ### 3.13.8

--- a/worker/subprojects/liburing.wrap
+++ b/worker/subprojects/liburing.wrap
@@ -3,11 +3,11 @@ directory = liburing-liburing-2.4
 source_url = https://github.com/axboe/liburing/archive/refs/tags/liburing-2.4.tar.gz
 source_filename = liburing-2.4.tar.gz
 source_hash = 2398ec82d967a6f903f3ae1fd4541c754472d3a85a584dc78c5da2fabc90706b
-patch_filename = liburing_2.4-1_patch.zip
-patch_url = https://wrapdb.mesonbuild.com/v2/liburing_2.4-1/get_patch
-patch_hash = 0dd939828da6a7c6d22c0f4db879405e74f9ac7153927c365642a28f871c92c2
-source_fallback_url = https://github.com/mesonbuild/wrapdb/releases/download/liburing_2.4-1/liburing-2.4.tar.gz
-wrapdb_version = 2.4-1
+patch_filename = liburing_2.4-2_patch.zip
+patch_url = https://wrapdb.mesonbuild.com/v2/liburing_2.4-2/get_patch
+patch_hash = 0435ae1012065fa96a22e1068c2e39e2f7c7c03b58a812774434a6e7455d7f20
+source_fallback_url = https://github.com/mesonbuild/wrapdb/releases/download/liburing_2.4-2/liburing-2.4.tar.gz
+wrapdb_version = 2.4-2
 
 [provide]
 dependency_names = liburing

